assets/js/modules/analytics/datastore/profiles.js

```diff
@@ -179,13 +200,15 @@ export const controls = {
 		return API.get( 'modules', 'analytics', 'profiles', {
 			accountID,
 			propertyID,
+		}, {
+			useCache: false,
```

Comment:
I agree it's a consideration, but I think we should stick to `true` as a default for now. The reasoning behind it is that caching by default is more sane to avoid aggressively requesting the same data. I _think_ verbosely specifying `false` for the case we always need fresh data is a better approach regarding performance and API quotas.

We're mostly passing `false` at the moment, but that is only because we haven't implemented datastore pieces yet that get regular insights - all of these should use the caching.